src/map_transition.js

```diff
@@ -78,11 +78,11 @@ function mapTransition (viewId, map, fitBoundsOptions) {
   map.once('moveend', showOverlays)
 
   function showOverlays () {
-    Object.keys(view.layerOpacity).forEach(function (layerId) {
+    Object.keys(view.layers).forEach(function (layerId) {
```

Comment:
layers are now defined in CMS not for opacity, but for visibility. opacity should now be taken from the map.